packages/eslint-config-checkout/index.js

```diff
@@ -1,30 +1,75 @@
 module.exports = {
   parser: "@typescript-eslint/parser",
+  plugins: ["@typescript-eslint", "formatjs"],
+  extends: [
+    "next",
+    "next/core-web-vitals",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:@typescript-eslint/recommended-requiring-type-checking",
+    "prettier",
+  ],
   parserOptions: {
-    tsconfigRootDir: "./",
-    project: ["./tsconfig.json"],
+    project: ["tsconfig.json"],
   },
-  plugins: ["@typescript-eslint"],
-  extends: ["prettier"],
   settings: {
     next: {
       rootDir: ["apps/*/", "packages/*/"],
     },
+    react: {
+      version: "999.999.999",
```

Comment:
Wait what? Why do we need to specify this version? 😅

Reply:
https://github.com/jsx-eslint/eslint-plugin-react/issues/1981
I'll add a comment xD

Reply:
Update: seems `"version": "detect"` now works. Not sure if that's related to some changes in eslint or what but thanks for making me check again.